cmd/telegraf/telegraf.go

```diff
@@ -267,6 +275,18 @@ func main() {
 		processorFilters = strings.Split(":"+strings.TrimSpace(*fProcessorFilters)+":", ":")
 	}
 
+	if *pprofAddr != "" {
+		go func() {
+			log.Printf(
+				"I! Starting pprof on %s. Open profiling tools page in browser: /debug/pprof",
```

Comment:
I think we have all the information we need to print the full url to this page, let have it write something like:
`I! Starting pprof HTTP server at http://localhost:6060/debug/pprof`  This way I can click on it or copy paste it more easily.